### ISSUE TITLE
HeatMap not work

### ISSUE THREAD

**Issue by talentAN** [zilliztech/arctern#416]
{
    "id": "1",
    "query": {
        "sql": "SELECT ST_Point (pickup_longitude, pickup_latitude) AS point, passenger_count AS w FROM global_temp.nyc_taxi WHERE (ST_Within (ST_Point (pickup_longitude, pickup_latitude), 'POLYGON ((-73.5 40.1, -73.5 41.1, -70.5 41.1, -70.5 40.1, -73.5 40.1))'))",
        "type": "heat",
        "params": {
            "width": 1024,
            "height": 896,
            "heat": {
                "bounding_box": [
                    -73.998427,
                    40.730309,
                    -73.954348,
                    40.780816
                ],
                "coordinate": "EPSG:4326",
                "map_scale": 10
            }
        }
    }
}

**Comment by talentAN** [comment]
以上sql在上个版本是可以跑的

**Comment by bigsheeper** [comment]
**The data in bounding box should be consistent with the polygon parameter in ST_Within, so the json should be like so:**
```
{
"id": "1",
"query": {
"sql": "SELECT ST_Point (pickup_longitude, pickup_latitude) AS point, passenger_count AS w FROM global_temp.nyc_taxi WHERE (ST_Within (ST_Point (pickup_longitude, pickup_latitude), 'POLYGON ((-73.5 40.1, -73.5 41.1, -70.5 41.1, -70.5 40.1, -73.5 40.1))'))",
"type": "heat",
"params": {
"width": 1024,
"height": 896,
"heat": {
"bounding_box": [-73.5, 40.1, -70.5, 41.1],
"coordinate": "EPSG:4326",
"map_scale": 10
}
}
}
}
```
**By running this json, I got the right result.**

**Comment by bigsheeper** [comment]
By the way, if you still want a picture without point, that's OK. This has been fixed by #420 , and you would get a blank picture.
